test/e2e/helloworld_test.go

```diff
@@ -136,6 +139,9 @@ func TestQueueSideCarResourceLimit(t *testing.T) {
 	} else {
 		t.Fatalf("Failed to get Service name from Revision label")
 	}
+	if val := revision.Status.ActualReplicas; val != 1 {
```

Comment:
Im wondering why we added this check to this test in particular (seems not particularly related?).